### PR TITLE
Fix typo.

### PR DESCRIPTION
**Pull request by aushack**
ares -> are.

### CODE REVIEW

**Comment by aushack** [comment]
Also postgres missing default port.

File: modules/exploits/multi/postgres/postgres_createlang.rb [hunk 1]
```diff
@@ -49,6 +49,10 @@ def initialize(info = {})
       ],
       'DefaultTarget' => 0,
       'DisclosureDate' => 'Jan 1 2016'))
+	  register_options(
```
Comment (wvu-r7) [inline modules/exploits/multi/postgres/postgres_createlang.rb]: I removed the hard tab here.

**Comment by wvu-r7** [comment]
https://github.com/rapid7/metasploit-framework/commit/7d44bd5ba40df21f9e171d03325318ffcac5fa70

File: modules/exploits/multi/postgres/postgres_createlang.rb (hunk 1 above)
Comment (aushack) [inline modules/exploits/multi/postgres/postgres_createlang.rb]: Oops yeah sorry, changed my text editor.

**Comment by wvu-r7** [comment]
# Release Notes

None needed.

**Pull request merged**